spec/features/test_page_spec.rb

```diff
@@ -3,4 +3,8 @@
     visit('/test')
     expect(page).to have_content "Test page"
   end
+  scenario 'page says hello chitter' do
+    visit('/')
+    expect(page).to have_content "Chitter"
```

Comment:
Similarly to testing the all method, this test could insert a row to the db at the beginning and then check the attributes of the tweet appear on the page